editors/sc-ide/CMakeLists.txt

```diff
@@ -350,21 +350,29 @@ if(APPLE)
                 #fixup_qt5_executable(\"${CMAKE_INSTALL_PREFIX}/SuperCollider/SuperCollider.app\")
                 " COMPONENT Runtime)
 
-
+elseif(CMAKE_GENERATOR MATCHES "Visual Studio")
+  set_property(TARGET SuperCollider PROPERTY RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/editors/sc-ide")
+  install(TARGETS SuperCollider
+      CONFIGURATIONS Debug;Release
+      DESTINATION "SuperCollider"
+      PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+  install(FILES ${translations}
+      CONFIGURATIONS Debug;Release
```

Comment:
wasapi, wdmks and dsound are not supported in the old portaudio that shipped with SC, and all three in fact hard to compile, especially in an MinGW environment. The current VS build I was using had all those apis built, but I was wondering if anybody ever used them. One price of the versatility is that you get an enormous list of devices when the server boosts, which is probably more confusing than helpful, and of course the build get's more fragile.
Do you agree that WMME and Asio are enough? In that case, building with ASIO or not would only depend on whether the person who builds SC adds the ASIO SDK in a sibling folder of the portaudio source (and dsound would have to be hardcoded to OFF to guard against building it on machines where directX is installed (80% of the attempts to build with DX support fail - only certain pa-dx-compiler-combinations work)).